nav2_dwb_controller/dwb_controller/src/dwb_controller.cpp

```diff
@@ -33,6 +33,8 @@ DwbController::DwbController()
 {
   RCLCPP_INFO(get_logger(), "Creating");
 
+  declare_parameter("controller_frequency", 20.0);
```

Comment:
We're changing the loop rate from 100ms to 50ms. Is this desired?

Reply:
Yes. This was the ROS 1 default.